packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -299,6 +303,66 @@ const assertNoImportedCssVariables = (
   }
 };
 
+/**
+ * Manipulates template literal so that it resembles
+ * CSS `property: value` declaration where
+ * - before: all the CSS _before_ the interpolation
+ * - after: all the CSS _after_ the interpolation
+ * @param expression
+ * @param quasi
+ */
+const moveCssPropertyInExpression = (
+  expression: t.ConditionalExpression,
+  quasi: t.TemplateElement,
+  nextQuasis: t.TemplateElement,
+  deleteValue = true
+): void => {
+  const conditionalPaths: ['consequent', 'alternate'] = ['consequent', 'alternate'];
+  const value = quasi.value.raw;
+  const re = /([^;/]+$)/; // everything after the last ';'
```

Comment:
What was `re` stand for?

Reply:
regex, but I was meant to rename it at the end